### PR TITLE
Featuredata add id column

### PR DESCRIPTION
**Pull request by DenverCoder544**
Always add ID column. Hidden by default,

image

### CODE REVIEW

File: bundles/framework/featuredata/view/FeatureDataContainer.jsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const createFeaturedataGrid = (features, selectedFeatureIds, showSelectedFirst,
 const createColumnSettings = (selectedFeatureIds, showSelectedFirst, sorting, visibleColumnsSettings) => {
     const { allColumns, visibleColumns, activeLayerPropertyLabels } = visibleColumnsSettings;
     return allColumns
-        .filter(key => !FEATUREDATA_DEFAULT_HIDDEN_FIELDS.includes(key) && visibleColumns.includes(key))
+        .filter(key => (!FEATUREDATA_DEFAULT_HIDDEN_FIELDS.includes(key) || key === ID_FIELD) && visibleColumns.includes(key))
```
Comment (ZakarFin) [inline bundles/framework/featuredata/view/FeatureDataContainer.jsx:103]: This is repeated so many times we should probably just add an export for something like `isFieldVisible(field, visibleColumns)` FeatureDataPluginHandler. Then we could comment on it why is ID_FIELD both in FEATUREDATA_DEFAULT_HIDDEN_FIELDS and handled separately?

File: bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { filterFeaturesByPropertyFilter } from '../../../mapping/mapmodule/oskar
 import { cleanFilter } from 'oskari-ui/components/FeatureFilter';
 
 export const FEATUREDATA_DEFAULT_HIDDEN_FIELDS = ['__fid', '__centerX', '__centerY', 'geometry'];
+export const ID_FIELD = '__fid';
```
Comment (ZakarFin) [inline bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js:10]: I'm not sure why these need to be separate but we can use the constant to make it clear it's the same field:
```suggestion
export const ID_FIELD = '__fid';
export const FEATUREDATA_DEFAULT_HIDDEN_FIELDS = [ID_FIELD, '__centerX', '__centerY', 'geometry'];
```

File: bundles/framework/featuredata/view/FeatureDataContainer.jsx (hunk 1 above)
Comment (DenverCoder544) [reply bundles/framework/featuredata/view/FeatureDataContainer.jsx]: Turns out we don't actually need the (historical) hidden_fields implementation for anything anymore. Removed that and it made things a whole lot simpler.

Also, didn't see the point of artificially cleaning out the id field from the list of properties in "select by properties"-dialog so selecting by id-field is now allowed.

File: bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js (hunk 2 above)
Comment (DenverCoder544) [reply bundles/framework/featuredata/plugin/FeatureDataPluginHandler.js]: this was more or less a historical remainder. Got rid of it now.

**Pull request merged**